source/lifecycle/client-compatibility.md

```diff
@@ -0,0 +1,28 @@
+# Client compatibility
```

Comment:
@nab-77 & @amyblais - Can I get your help to draft the client compatibility content we want published on this page, please?

Reply:
For reference: Desktop app v5.8.0 (May 16th release) was tested to be compatible with v9.5.0 and newer releases (and v8.1.0 ESR which is now out support) on Linux/Mac/Windows.